src/main/java/database/helper/Settings.java

```diff
@@ -83,4 +82,29 @@ Path getFilePath(String directoryName) throws CreationDatabaseException {
             throw new CreationDatabaseException("Directory cannot be created: " + e.getMessage());
         }
     }
+
+    private Properties loadProperties() {
+        if (propertyFileName == null) {
+            return null;
+        }
+
+        try (var input = Settings.class.getClassLoader().getResourceAsStream(propertyFileName)) {
+            if (input == null) {
+                return null;
+            }
+            properties.load(input);
+            return properties;
+        } catch (IOException e) {
+            LOG.error("Properties file cannot be loaded: {}", e.getMessage());
+            return null;
+        }
+    }
+
+    private String getProperty(String propertyName, String defaultValue) {
+        Properties properties = loadProperties();
```

Comment:
Consider declaring `getProperty` method before `loadProperties` to simplify navigation between methods.